aspnet/mvc/overview.rst

```diff
@@ -1,8 +1,317 @@
+
+Binding Model
+#############
+
+Sometimes it may be worthwhile to create a type specifically for use with `model binding`_. These types are typically just data containers with no behavior.
+
+.. note:: Using a binding model (or viewmodel) is recommended to avoid allowing a malicious user to set properties on the model that are not included in the UI, but are exposed via model binding.
+
+API Model
+#########
+
+If your application exposes an API, the format of the data you expose to clients may be separated from your app's internal domain model by defining custom API model types. This allows you to change your internal model types without impacting clients that may be using your exposed APIs.
+
+Controller Responsibilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Controllers are the components that handle user interaction, work with the model, and ultimately select a view to render. In an MVC application, the view only displays information; the controller handles and responds to user input and interaction. In the MVC pattern, the controller is the initial entry point, and is responsible for selecting which model types to work with and which view to render (hence its name - it controls how the app responds to a given request).
+
+.. note:: Controllers should not be overly complicated by too many responsibilities. To keep controller logic from becoming overly complex, use the `Single Responsibility Principle <http://deviq.com/single-responsibility-principle/>`_ to push business logic out of the controller and into the domain model.
+
+.. tip:: If you find that your controller actions frequently perform the same kinds of actions, you can follow the `Don't Repeat Yourself principle <http://deviq.com/don-t-repeat-yourself/>`_ by moving these common actions into `filters`_.
+
+View Responsibilities
+^^^^^^^^^^^^^^^^^^^^^
+
+Views are responsible for presenting content through the user interface. There should be minimal logic within views, and any logic in them should relate to presenting content. If you find the need to perform a great deal of logic in view files in order to display data from a complex model, consider using a :doc:`View Component </mvc/views/view-components>`, ViewModel, or view template to simplify the view.
+
+What is ASP.NET Core MVC
+------------------------
+
+The ASP.NET Core MVC framework is a lightweight, open source, highly testable presentation framework optimized for use with ASP.NET Core.
+
+ASP.NET Core MVC provides a patterns-based way to build dynamic websites that enables a clean separation of concerns. It gives you full control over markup, supports TDD-friendly development and uses the latest web standards.
+
+Features
```

Comment:
My earlier draft was more paragraph-form; I took the list form from your earlier draft, @danroth27 . I think, having seen both, that your list format works, though, since there are so many to discuss. It makes it easy for the reader to navigate and jump to a particular thing they see that is of interest.